theme/static/js/hs_resource_table.js

```diff
@@ -166,9 +271,44 @@ $(document).ready(function () {
 
     updateLabelLists();
     updateLabelCount();
-
 });
 
+function delete_multiple_resources_ajax_submit(indexes) {
+    var calls = [];
+
+    // Submit all delete requests asynchronously
+    for (var i = 0; i < indexes.length; i++) {
+        var form = $(indexes[i]).find("form[data-form-type='delete-resource']");
+        const row = $(indexes[i]);  // Needs to be a constant so the value doesn't change during the asynchronous calls
+        var datastring = $(form).serialize();
+        var url = $(form).attr("action");
+
+        $("html").css("cursor", "progress");
+
+        calls.push(
+            $.ajax({
+                type: "POST",
+                url: url,
+                datastring: datastring,
+                dataType: "html",
+                success: function () {
+                    resourceTable.row(row).remove();  // Delete row from the table
+                },
+                error: function (XMLHttpRequest, textStatus, errorThrown) {
+                    // Unused for now
```

Comment:
@mauriel133 I think the error messages should be printed to console if ajax responds with an error.